app/build.gradle

```diff
@@ -40,7 +40,7 @@ android {
         applicationId "com.twilio.video.app"
 
         minSdkVersion 16
-        targetSdkVersion 28
+        targetSdkVersion 29
```

Comment:
I don't think we can target 29 at this time due to https://issues.corp.twilio.com/browse/AHOYAPPS-213